lib/systems/include/CASESystem.tpp

```diff
@@ -200,71 +200,71 @@ DrivetrainCommand_s CASESystem<message_queue>::evaluate(
 
     // send these out at the send period
 
-    if ((tick.millis - last_controller_pt1_send_time_) >= (controller_send_period_ms_))
-    {
-        enqueue_matlab_msg(msg_queue_, res.controllerBus_controller_initia);
-        enqueue_matlab_msg(msg_queue_, res.controllerBus_controller_norm_p);
-        enqueue_matlab_msg(msg_queue_, res.controllerBus_controller_pow_pn);
-        enqueue_matlab_msg(msg_queue_, res.controllerBus_controller_tcs_to);
-        enqueue_matlab_msg(msg_queue_, res.controllerBus_controller_regen_);
-        enqueue_matlab_msg(msg_queue_, res.controllerBus_controller_pid__p);
-
-        last_controller_pt1_send_time_ = tick.millis;
-    }
+    // if ((tick.millis - last_controller_pt1_send_time_) >= (controller_send_period_ms_))
```

Comment:
Consider deleting this if unnecessary and replacing it with a note. We can always reference code back from git.